TRACK_README.md

```diff
@@ -1,11 +1,58 @@
 # Exercism {{LANGUAGE}} Track
 
+![build status](https://travis-ci.org/exercism/xslug.svg?branch=master)
+
 Exercism exercises in {{LANGUAGE}}.
 
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading the general Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
```

Comment:
Haha, yes. Both! "getting involved" first, and then the CONTRIBUTING after, perhaps saying that it has a bunch of topics, so scan the table of contents for things relevant to what you're trying to do.